File: jscomp/test/lib_js_test.js
'use strict';

Mt = require("./mt.js");
Block = require("../../lib/js/block.js");

console.log(JSON.stringify(--[[ :: ]][
          1,
          --[[ :: ]][
            2,
            --[[ :: ]][
              3,
              --[[ [] ]]0
            ]
          ]
        ]));

console.log("hey");

suites_000 = --[[ tuple ]][
  "anything_to_string",
  (function (param) do
      return --[[ Eq ]]Block.__(0, [
                "3",
                String(3)
              ]);
    end end)
];

suites = --[[ :: ]][
  suites_000,
  --[[ [] ]]0
];

Mt.from_pair_suites("Lib_js_test", suites);

exports.suites = suites;
--[[  Not a pure module ]]

File: jscomp/test/lib_js_test.lua
__console = {log = print};

Mt = require "..mt";
Block = require "......lib.js.block";

__console.log(__JSON.stringify(--[[ :: ]]{
          1,
          --[[ :: ]]{
            2,
            --[[ :: ]]{
              3,
              --[[ [] ]]0
            }
          }
        }));

__console.log("hey");

suites_000 = --[[ tuple ]]{
  "anything_to_string",
  (function(param) do
      return --[[ Eq ]]Block.__(0, {
                "3",
                __String(3)
              });
    end end)
};

suites = --[[ :: ]]{
  suites_000,
  --[[ [] ]]0
};

Mt.from_pair_suites("Lib_js_test", suites);

exports = {};
exports.suites = suites;
return exports;
--[[  Not a pure module ]]
